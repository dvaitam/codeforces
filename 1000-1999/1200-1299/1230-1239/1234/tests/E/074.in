17 9
10 10 7 14 11 6 7 1 16
