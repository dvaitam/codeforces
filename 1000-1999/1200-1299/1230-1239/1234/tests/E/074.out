40 38 38 38 38 48 50 36 36 41 49 37 37 47 39 26 40
